parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -222,18 +217,29 @@ setupCNIDirs() {
     chmod 755 $CNI_CONFIG_DIR
 }
 
-# For CNI/AzureCNI, we want to use the untar azurecni reference first. And if that doesn't exist on the vhd does the tgz?
-# And if tgz is already on the vhd then just untar into CNI_BIN_DIR
-# Latest VHD should have the untar, older should have the tgz. And who knows will have neither.
+
+# Reference CNI plugins is used by kubenet and the loopback plugin used by containerd 1.0 (dependency gone in 2.0)
+# The version used to be deteremined by RP/toggle but are now just hadcoded in vhd as they rarely change and require a node image upgrade anyways
+# Latest VHD should have the untar, older should have the tgz. And who knows will have neither. 
 installCNI() {
-    CNI_TGZ_TMP=${CNI_PLUGINS_URL##*/} # Use bash builtin ## to remove all chars ("*") up to the final "/"
+    #how do we keep this in sync with whats in AgentBaker/parts/linux/cloud-init/artifacts/components.json
+    # and how is that different from AgentBaker/vhdbuilder/packer/components.json 
```

Comment:
simplification incoming, but both the components.json files in the differents dirs you mention are the exact same (the one in vhdbuilder is actually just a symlink to the other...)

Reply:
okay getting rid of second line